### PR TITLE
ci: set up closing-inactive-issues in GitHub Action (#415)

### PR DESCRIPTION
**Pull request by vvatanabe**
**Describe the change**
Managing neglected issues manually proves to be an arduous issue. It is desirable to have a bot in place to automate the closure of such issues.

**Describe your solution**
Set up closing-inactive-issues in GitHub Action.
Refs: https://docs.github.com/en/github-ae@latest/actions/managing-issues-and-pull-requests/closing-inactive-issues

**Tests**
None

**Additional context**
Issue: #415

### CODE REVIEW

File: .github/workflows/close-inactive-issues.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      - uses: actions/stale@v5
+        with:
+          days-before-issue-stale: 30
+          days-before-issue-close: 14
+          stale-issue-label: "stale"
+          exempt-issue-labels: 'bug,enhancement'
+          stale-issue-message: "This issue is stale because it has been open for 30 days with no activity."
+          close-issue-message: "This issue was closed because it has been inactive for 14 days since being marked as stale."
+          days-before-pr-stale: -1
+          days-before-pr-close: -1
```
Comment (vvatanabe) [inline .github/workflows/close-inactive-issues.yml:22]: `-1` is set so this action does not apply to PR. We don't have to use it for PR, right?

File: .github/workflows/close-inactive-issues.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/stale@v5
+        with:
+          days-before-issue-stale: 30
+          days-before-issue-close: 14
+          stale-issue-label: "stale"
+          exempt-issue-labels: 'bug,enhancement'
```
Comment (vvatanabe) [inline .github/workflows/close-inactive-issues.yml:18]: Issues labeled bug or enhancement are not target for this Action.

File: .github/workflows/close-inactive-issues.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+name: Close inactive issues
+on:
+  schedule:
+    - cron: "30 1 * * *"
```
Comment (vvatanabe) [inline .github/workflows/close-inactive-issues.yml:4]: It will be executed daily at 1:30 (UTC).

**Comment by vvatanabe** [comment]
@sashabaranov I gave it a shot. What do you think?

**Comment by vvatanabe** [comment]
@sashabaranov I want to try adding this GitHub Action. Would it be okay to merge it?

**Review by sashabaranov — APPROVED** [review]
Thank you for the PR and sorry for taking so much time at looking into it!

**Pull request merged**